sample.env

```diff
@@ -368,6 +368,8 @@ HOST_BLOB_STORAGE_DIRECTORY=./local_blob_storage
 ## Maximum retry limit for performing firmware upgrades
 FW_UPGRADE_MAX_RETRY_LIMIT=3
 
+FIRMWARE_MANAGER_UPGRADE_SCHEDULER_ENDPOINT=http://${DOCKER_HOST_IP}:8089
```

Comment:
Port 8089 is defined in the docker-compose-addons as the exposed port, hence the default.